gno.land/cmd/gnoland/start.go

```diff
@@ -174,283 +189,153 @@ func execStart(c *startCfg, io *commands.IO) error {
 	}
 
 	if err := gnoNode.Start(); err != nil {
 		return fmt.Errorf("error in start node: %w", err)
 	}
 
-	// run forever
 	osm.TrapSignal(func() {
 		if gnoNode.IsRunning() {
 			_ = gnoNode.Stop()
 		}
 	})
 
-	select {} // run forever
-}
-
-// getTxEventStoreConfig constructs an event store config from provided user options
-func getTxEventStoreConfig(c *startCfg) (*eventstorecfg.Config, error) {
-	var cfg *eventstorecfg.Config
-
-	switch c.txEventStoreType {
-	case file.EventStoreType:
-		if c.txEventStorePath == "" {
-			return nil, errors.New("unspecified file transaction indexer path")
-		}
-
-		// Fill out the configuration
-		cfg = &eventstorecfg.Config{
-			EventStoreType: file.EventStoreType,
-			Params: map[string]any{
-				file.Path: c.txEventStorePath,
-			},
-		}
-	default:
-		cfg = eventstorecfg.DefaultEventStoreConfig()
-	}
-
-	return cfg, nil
+	// Run forever
+	select {}
 }
 
-// Makes a local test genesis doc with local privValidator.
-func makeGenesisDoc(
-	pvPub crypto.PubKey,
-	chainID string,
-	genesisBalancesFile string,
-	genesisTxs []std.Tx,
-) (*bft.GenesisDoc, error) {
+func generateGenesisFile(genesisFile string, pk crypto.PubKey, c *startCfg) error {
 	gen := &bft.GenesisDoc{}
-
 	gen.GenesisTime = time.Now()
-	gen.ChainID = chainID
+	gen.ChainID = c.chainID
 	gen.ConsensusParams = abci.ConsensusParams{
 		Block: &abci.BlockParams{
 			// TODO: update limits.
-			MaxTxBytes:   1000000,  // 1MB,
-			MaxDataBytes: 2000000,  // 2MB,
-			MaxGas:       10000000, // 10M gas
-			TimeIotaMS:   100,      // 100ms
+			MaxTxBytes:   1_000_000,  // 1MB,
+			MaxDataBytes: 2_000_000,  // 2MB,
+			MaxGas:       10_0000_00, // 10M gas
+			TimeIotaMS:   100,        // 100ms
 		},
 	}
+
 	gen.Validators = []bft.GenesisValidator{
 		{
-			Address: pvPub.Address(),
-			PubKey:  pvPub,
+			Address: pk.Address(),
+			PubKey:  pk,
 			Power:   10,
 			Name:    "testvalidator",
 		},
 	}
 
-	// Load distribution.
-	balances, err := loadGenesisBalances(genesisBalancesFile)
+	// Load balances files
+	balances, err := gnoland.LoadGenesisBalancesFile(c.genesisBalancesFile)
 	if err != nil {
-		return nil, fmt.Errorf("unable to load genesis balances, %w", err)
+		return fmt.Errorf("unable to load genesis balances file %q: %w", c.genesisBalancesFile, err)
 	}
 
-	// Load initial packages from examples.
+	// Load examples folder
+	examplesDir := filepath.Join(c.gnoRootDir, "examples")
 	test1 := crypto.MustAddressFromString("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5")
-	txs := []std.Tx{}
-
-	// List initial packages to load from examples.
-	pkgs, err := gnomod.ListPkgs(filepath.Join("..", "examples"))
-	if err != nil {
-		panic(fmt.Errorf("listing gno packages: %w", err))
+	examplePkgs := gnoland.PackagePath{
+		Path:    examplesDir,
+		Creator: test1,
+		Fee:     std.NewFee(50000, std.MustParseCoin("1000000ugnot")),
+		Deposit: nil,
 	}
 
-	// Sort packages by dependencies.
-	sortedPkgs, err := pkgs.Sort()
+	pkgsTxs, err := examplePkgs.Load()
```

Comment:
This section is confusing. Consider using more descriptive variable names or simplifying the code to enhance clarity.